### ISSUE TITLE
Remove pkg_mgr & pkg_rm variables from distributed os.freebsd.conf

### ISSUE THREAD

**Issue by markonjezic** [extremeshok/clamav-unofficial-sigs#97]
If someone installs clamav-unofficial-sigs from source and not from the FreeBSD ports, when attempting to deinstall the script using "remove-script" command he will get a message to instead use FreeBSD package manager to remove the script and the script will not uninstall itself. This is obviously wrong. Those two pkg_mgr and pkg_rm variables that trigger this behavior should be removed from default distributed configuration file. And since I am current FreeBSD port maintainer, I will add those two pkg_* variables in the port itself. This way both situations: installation from source and installation from FreeBSD ports will act accordingly when attempting to remove the script.

**Comment by IdahoPL** [comment]
Why would anyone install clamav-unofficial-sigs on FreeBSD not from ports? Anyway, even if someone would like to install it from source he must patch it anyway, so he can remove pkg_* variables from config file if he doesn't need them.

**Comment by markonjezic** [comment]
@IdahoPL Why wouldn't someone install it from the source? You would be amazed at some of the things that people do, but in this case one scenario is when someone wants to install a newer version of the script than the one that's available from the ports tree. Next you go on saying that the one who installed it from the source is responsible to "patch it away". Those variables are not documented anywhere and average user won't be even aware that he has to remove them.

I don't see those pkg_* variables in any of the distributed os.conf files for various Linux distributions. If we follow your logic, then configuration file for CentOS/RHEL should have pkg_* variables that point to the YUM package manager, Debian/Ubuntu should point to the APT-GET, etc. And this is clearly wrong and that's why you won't see those directives in any of those files. The original script from Bill Landry did not have those variables defined in the configuration file for a reason.

**Comment by IdahoPL** [comment]
Could you show me where Bill Landry (or anyone else beside you) claims that this idea is "clearly wrong"?

**Comment by markonjezic** [comment]
@IdahoPL I'm not going to waste my time arguing with you and will leave this issue up to the current script maintainer. It's a pity that original script author did not have public development repository, so that you could see for yourself. And while I'm not at all related to the original implementation of the pkg_* variables, I was in contact with Bill Landry and you can see my name in original changelog file.

Here's an excerpt from the original changelog file, which shows that those two variables are meant to be used by package and port maintainers.

```
   - Provided two variables that package and port maintainers can use in
     order to prevent the script from removing itself with the '-r' flag
     if the script was installed via a package manager like yum, apt, pkg,
     etc.  The script will instead provide feedback to the user about how
     to uninstall the package.
```

Bottom line is this, everyone who installs this script from source on FreeBSD won't be able to uninstall it properly. Such problem does not exist when installing on Linux distributions.

**Comment by IdahoPL** [comment]
@markonjezic Ok, you're right, I rest my case.

**Comment by extremeshok** [comment]
Fixed with : 4d5be3a5d094167252893fb1607b62d20eb58709

**Comment by extremeshok** [comment]
thank you guys
